modules/UI/videolayout/RemoteVideo.js

```diff
@@ -576,6 +576,9 @@ RemoteVideo.prototype.addRemoteStreamElement = function (stream) {
 
     if (!isVideo) {
         this._audioStreamElement = streamElement;
+
+        // for the volume slider
```

Comment:
I will change the comment! This covers the scenario where the remote video menu is created but the audio stream is not attached yet. This will result in not having the volume slider in the menu and when the audio stream is attached we should update the menu in order to add the volume slider.